server/base/src/test/java/org/apache/accumulo/server/conf/AccumuloConfigurationIsPropertySetTest.java

```diff
@@ -0,0 +1,327 @@
+    assertTrue(shouldBeSet.stream().allMatch(mapContainsProp));
+    assertTrue(shouldNotBeSet.stream().noneMatch(mapContainsProp));
+  }
+
+  @Test
+  public void configurationCopy() {
+    Set<Property> shouldBeSet = Set.of(TABLE_BLOOM_SIZE, GC_PORT);
+    Set<Property> shouldNotBeSet = Sets.difference(shouldBeSet, ALL_PROPERTIES);
+
+    // set up object
+    ConfigurationCopy configurationCopy = new ConfigurationCopy();
+    shouldBeSet.forEach(property -> configurationCopy.set(property, "foo"));
+
+    verifyProps(configurationCopy, shouldBeSet, shouldNotBeSet);
+
+    testPropertyIsSetImpl(configurationCopy, shouldBeSet, shouldNotBeSet);
+  }
+
+  @Test
+  public void defaultConfiguration() {
+    Set<Property> shouldBeSet = Collections.emptySet();
+    Set<Property> shouldNotBeSet = new HashSet<>(ALL_PROPERTIES);
+
+    DefaultConfiguration defaultConfiguration = DefaultConfiguration.getInstance();
+
+    // we don't expect isPropertySet to be true for any prop since props cannot be set by the user
+    // for DefaultConfiguration
+    testPropertyIsSetImpl(defaultConfiguration, shouldBeSet, shouldNotBeSet);
+  }
+
+  @Test
+  public void namespaceConfiguration() {
+    InstanceId iid = InstanceId.of(UUID.randomUUID());
+    ZooPropStore propStore = createMock(ZooPropStore.class);
+    propStore.registerAsListener(anyObject(), anyObject());
+    expectLastCall().anyTimes();
+    ZooReaderWriter zrw = createMock(ZooReaderWriter.class);
+    ServerContext context = getMockContextWithPropStore(iid, zrw, propStore);
+    ConfigurationCopy parent = new ConfigurationCopy(
+        Map.of(TABLE_BLOOM_SIZE.getKey(), TABLE_BLOOM_SIZE.getDefaultValue()));
+    reset(propStore);
+    NamespaceId NSID = NamespaceId.of("namespace");
+    var nsPropStoreKey = NamespacePropKey.of(iid, NSID);
+    expect(propStore.get(eq(nsPropStoreKey))).andReturn(new VersionedProperties(123, Instant.now(),
+        Map.of(Property.INSTANCE_SECRET.getKey(), "sekrit"))).anyTimes();
+    propStore.registerAsListener(eq(nsPropStoreKey), anyObject());
+    expectLastCall().anyTimes();
+    replay(propStore, context);
+
+    NamespaceConfiguration namespaceConfiguration =
+        new NamespaceConfiguration(context, NSID, parent);
+
+    Set<Property> shouldBeSet = Set.of(TABLE_BLOOM_SIZE, INSTANCE_SECRET);
+    Set<Property> shouldNotBeSet = Sets.difference(shouldBeSet, ALL_PROPERTIES);
+
+    verifyProps(namespaceConfiguration, shouldBeSet, shouldNotBeSet);
+
+    testPropertyIsSetImpl(namespaceConfiguration, shouldBeSet, shouldNotBeSet);
+  }
+
+  @SuppressFBWarnings(value = {"PATH_TRAVERSAL_IN"}, justification = "path provided by test")
+  @Test
+  public void siteConfiguration() throws IOException {
+
+    Set<Property> shouldBeSet =
+        Set.of(INSTANCE_ZK_HOST, INSTANCE_SECRET, MANAGER_BULK_TSERVER_REGEX);
+    Set<Property> shouldNotBeSet = Sets.difference(shouldBeSet, ALL_PROPERTIES);
```

Comment:
This needs the same change as mentioned elsewhere

```suggestion
    Set<Property> shouldNotBeSet = Sets.difference(ALL_PROPERTIES, shouldBeSet);
```